qemu/tests/cfg/multi_host.cfg

```diff
@@ -37,15 +37,20 @@
                 #Time when start migration
                 - after_login_vm:
                     paused_after_start_vm = no
+                    need_to_login = yes
+                    preprocess_env = no
                 - early_boot_vm:
                     no measure_migration_speed
                     login_timeout = 420
                     paused_after_start_vm = yes
+                    preprocess_env = yes
                     variants:
                         - timeout_0:
                             start_migration_timeout = 0
                         - timeout_6:
                             start_migration_timeout = 6
+                        - timeout_random:
+                            start_migration_timeout = random
```

Comment:
Why set range between 5 and 20?
There's a case which has 6 second timeout defined by timeout_6.
And random.randint() maybe return 6.
So we maybe test the same situation in two cases.